builders/integrations/indexers/moralis.md

```diff
@@ -0,0 +1,182 @@
+
+=== "npm"
+
+    ```bash
+    npm install moralis
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn add moralis
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm add moralis
+    ```
+
+=== "pip"
+
+    ```bash
+    pip install moralis
+    ```
+
+Once the SDK is ready, you can leverage it to query Moralis APIs. Here's an example of how to interact with the EVM API using a JSON RPC interface:
+
+=== "Javascript"
+
+    ```js
+    import Moralis from 'moralis';
+
+        try {
+          await Moralis.start({
+            apiKey: "INSERT_YOUR_API_KEY"
+    });
+
+          const response = await Moralis.EvmApi.block.getBlock({
+            "chain": "0x504",
+            "blockNumberOrHash": "1"
+    });
+
+          console.log(response.raw);
+    } catch (e) {
+          console.error(e);
+    }
+    ```
+
+=== "Python"
+
+    ```python
+    from moralis import evm_api
+
+    api_key = "INSERT_YOUR_API_KEY"
+
+    params = {
+          "chain": "moonbeam",
+          "block_number_or_hash": "1"
+    }
+
+    result = evm_api.block.get_block(
+          api_key=api_key,
+          params=params,
+    )
+
+        print(result)
+    ```
+
+
+## Stream API {: #stream-api }
+
+Moralis Streams API is a tool for developers to listen to and react to events happening on blockchains in real time. This API lets you listen for specific events, such as a new transaction being added to a block, a particular smart contract function being called, or an NFT being transferred and delivering the event via Webhook.
+
+Head over to the Streams page within Moralis and click **Create a new stream** to get started.
+
+![stream API page](/images/builders/integrations/indexers/moralis/moralis-2.webp)
+
+To get started quickly, let's choose a predefined template here. In this example, we're interested in xcDOT transactions. Select the "Contract Activity" template. This will allow us to easily listen for events related specifically to xcDOT.
+
+![template selection page](/images/builders/integrations/indexers/moralis/moralis-3.webp)
+
+On the next page, enter the following details: 
+
+- Smart Contract Address for [xcDot](https://moonscan.io/token/0xffffffff1fcacbd218edc0eba20fc2308c778080): 0xFfFFfFff1FcaCBd218EDc0EbA20Fc2308C778080
+- Network: Select Moonbeam from the available options.
+Moralis supports listening to events on Moonbeam, Moonriver, and Moonbase Alpha.  
+
+![details input page](/images/builders/integrations/indexers/moralis/moralis-4.webp)
+
+Next, customize the events you want to receive. For this example, choose "All contract events." 
+
+You may also test the stream to confirm it captures the needed data.  
+
+![event selection and stream test page](/images/builders/integrations/indexers/moralis/moralis-5.webp)
+
+Next, you can connect the stream to an endpoint to receive updates from Moralis.
+
+![webhook setup page](/images/builders/integrations/indexers/moralis/moralis-6.webp)
+
+All set. Once you activate your stream, you'll see a screen like this:
```

Comment:
to keep our documentation accessible, you should describe what the screen looks like instead of relying solely on the image. so instead say "Once you activate your stream, it will be added to your list of streams on your dashboard"